src/sentry/static/sentry/app/views/settings/projectPlugins/details.jsx

```diff
@@ -26,6 +28,31 @@ import {DEPRECATED_PLUGINS} from './constants';
  *    PluginsStore
  */
 class ProjectPluginDetails extends AsyncView {
+  componentDidUpdate(prevProps) {
+    if (prevProps.params.pluginId !== this.props.params.pluginId) {
+      this.recordDetailsViewed();
+    }
+  }
+  componentDidMount() {
+    this.recordDetailsViewed();
+  }
+
+  recordDetailsViewed() {
+    const {projectId, pluginId} = this.props.params;
+
+    trackIntegrationEvent(
+      {
+        eventKey: 'integrations.details_viewed',
+        eventName: 'Integrations: Details Viewed',
+        integration: pluginId,
+        integration_type: 'plugin',
+        view: 'plugin_details',
+        projectId,
```

Comment:
is this supposed to be camel case? Looks like we are doing `project_id` in other places. Also just noticing now that camel case for `eventKey` and `eventName` but snake case for everything else.

Reply:
@MeredithAnya Good catch, we should use `project_id` as fields should be snake case. I think the reason we use camel case for `eventKey` and `eventName` is because they are special fields and don't actually show up in our event data. `eventKey` maps to `type` for reload and `eventName` maps to `event` in Amplitude.